docs/repos/git/ignore-files.md

```diff
@@ -124,13 +124,13 @@ During development it's convenient to stop tracking file changes to a file commi
 customizing settings or configuration files that are part of your project source for your own work environment.
 
 <pre style="color:white;background-color:black;font-family:Consolas,Courier,monospace;padding:10px">
-&gt; git update-index --assume-unchanged <font color="#b5bd68">&lt;file&gt;</font>
+&gt; git update-index --skip-worktree <font color="#b5bd68">&lt;file&gt;</font>
 </pre>
 
 Resume tracking files with:
 
 <pre style="color:white;background-color:black;font-family:Consolas,Courier,monospace;padding:10px">
-&gt; git update-index --no-assume-unchanged <font color="#b5bd68">&lt;file&gt;</font>
+&gt; git update-index --no-skip-worktree <font color="#b5bd68">&lt;file&gt;</font>
```

Comment:
Thanks for the input! It looks like you missed removing the first part of the old command `--no`

Reply:
No, no. There is no error. 

https://git-scm.com/docs/git-update-index#Documentation/git-update-index.txt---no-skip-worktree

Reply:
I apologize! Must have haven't had my coffee yet in my first review! ;) Sorry for the delay and thank you again for the input! Merging and you should see it live within 24 hours